src/api/types/personoversiktTypes.ts

```diff
@@ -6,6 +13,8 @@ export interface PersonOversiktStatusDTO {
   motebehovUbehandlet: boolean | null;
   moteplanleggerUbehandlet: boolean | null;
   oppfolgingsplanLPSBistandUbehandlet: boolean | null;
+  dialogmotekandidat?: boolean;
```

Comment:
Hvis disse to feltene alltid er med i dtoen fra APIet tror jeg det blir riktigere med 
```
dialogmotekandidat: boolean | undefined;
motestatus: MoteStatusType | undefined;
```
(altså at de ikke er optional)

Reply:
Endret til `| undefined`